### PR TITLE
Major release candidate - v1.0

### PR DESCRIPTION
**Pull request by mjbrender**
* moved to sub-folders to organize types of docs
* let go of the release notes idea given it's not adopted - we note major projects on our website: http://basho.com/community/
* Updates to the labeling strategy based on Docker's awesome work
* Minor updates to CM details

### CODE REVIEW

**Comment by mjbrender** [comment]
Welcoming feedback! Tagging a few I think will want to check it out: @basho-labs/taishi-members @danieldreier @laurenrother @christophermancini

**Comment by mjbrender** [comment]
Note: This will also include changing the repo name to `the-basho-community` to represent our broader focus. You'll notice that syntax in the updates :+1:

**Comment by DeadZen** [comment]
+1 Matt

File: README.md [hunk 1]
```diff
@@ -1,49 +1,81 @@
...
+We've introduced the idea of "The Basho Community" as a releasable product. Why not approach community development like you would code? What if there were scheduled "releases" comprised of new "features" and "bug fixes"? 
 
-Each subsequent release will represent the evolution of the Community: new docs will have been written; new talks will have been given; new slide decks will have been created; process flaws will have been unearthed and rectified. All of this moves the community forward and makes it a better place to work and play.
- 
-Finally, this will be a way for us to track the progression of the community over time and build the community in a more collaborative, transparent way. There are people all over the world contributing to the growth of Riak in endless ways. This is an attempt at capturing and showcasing that growth.
+The idea is simple: each few months we cut a "new release" of the Basho Community. A release will be comprised of updates to **how we communicate**. The more detailed conversation can occur as [Issues](https://github.com/basho-labs/the-riak-community/issues). It's that simple.
```
Comment (laurenrother) [inline README.md]: 'each' should really be 'every'

File: README.md [hunk 2]
```diff
@@ -1,49 +1,81 @@
...
 
-### What Should Be Part of a Riak Community Release? 
+The goal is to periodically tag and release "versions" of The Basho Community. This tagging will signify specific objectives, both internal to Basho and external in our Community. 
 
-For the purposes of this project, a community release can include anything that's not code specific to Riak products, though work related to code (e.g. releasing a new client library definitely counts).  Anything that contributes to the advancement of the community as a whole. A partial list of examples:
+Each subsequent release will represent the evolution of the Community: new ways to share our code, communicate clearly and process flaws will have been unearthed and rectified. All of this moves the community forward and makes it a better place to work and play.
```
Comment (laurenrother) [inline README.md]: I'm a huge fan of the Oxford comma. I know not everyone is, so just pointing it out here.

File: README.md [hunk 3]
```diff
@@ -1,49 +1,81 @@
...
-2. Someone with commit rights to the repo will close the PR and you and your contribution will be added to the forthcoming Riak Community Release.
+1. Open an [Issue](https://github.com/basho-labs/the-riak-community/issues) as you have ideas for community improvement. These may be as general as requests for visibility ([example](https://github.com/basho-labs/the-riak-community/issues/69)) or as specific as request for coordinated improvements ([example](https://github.com/basho-labs/the-riak-community/issues/71))
+2. Open a [pull request](https://github.com/basho-labs/the-riak-community/pull/new/master) with details on your ideas for improvement or corrections to existing information
+3. Someone with commit rights to the repo will discuss the details over the PR
+
+#### Labels 
+
+You'll notice a detailed list of labels on our issues here (inspired by Docker). The major topics are broken out into 3 buckets: 
+
+* **Capitalized** - is manually applied labels that indicate an important status
```
Comment (laurenrother) [inline README.md]: I would maybe remove the 'to be' verb from these entirely, so:
**Capitalized** - manually-applied labels indicating an important status
**requests** - track requests for types of projects, from a how-to guide to a presentation
**status** gets people's attention and clarifies who's running with what task or project

File: README.md (hunk 2 above)
Comment (mjbrender) [inline README.md]: Where would it go @laurenrother? I'm still learning my Oxford style guidelines :smile:

File: README.md (hunk 3 above)
Comment (mjbrender) [inline README.md]: done :+1:

File: README.md (hunk 1 above)
Comment (mjbrender) [inline README.md]: done :+1:

File: README.md [hunk 4]
```diff
@@ -1,49 +1,81 @@
...
+## License
 
-## Thanks
-This project began with [Mark Phillips](https://twitter.com/pharkmillups) and it was quiet for a few years after that. I would like to keep it in mind as we reflect on the next steps for the Open Source ecosystem that Riak is a part of. Please send your thoughts to me, [@mjbrender on Twitter](https://twitter.com/mjbrender) or to [mbrender@basho.com](mailto:mbrender@basho.com).
+Feel free to use this format in your own community. 
 
-We'll see how this goes. It might fail miserably, but I don't think it'll be a total disaster. Also, if you think this idea is ruining the internet, or if there is a way to do this more effectively, please share with us.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
```
Comment (laurenrother) [inline README.md]: Is there a link to the license? Sometimes I can't tell...

**Comment by laurenrother** [comment]
I made some tiny comments (over a couple days because #life), but overall: :+1: :candy: :dancer:

**Comment by mjbrender** [comment]
@laurenrother I merged yours in and rebased them in. Link to license now too. Thanks!

**Comment by danieldreier** [comment]
+1

**Pull request merged**